Index: src/contracts/dependencies/openzeppelin/contracts/Strings.sol
===================================================================
--- src/contracts/dependencies/openzeppelin/contracts/Strings.sol
+++ src/contracts/dependencies/openzeppelin/contracts/Strings.sol
@@ -26,8 +26,9 @@
         }
         bytes memory buffer = new bytes(digits);
         while (value != 0) {
             digits -= 1;
+            // forge-lint: disable-next-line(unsafe-typecast)
             buffer[digits] = bytes1(uint8(48 + uint256(value % 10)));
             value /= 10;
         }
         return string(buffer);
