===================================================================
--- src/contracts/interfaces/IPoolAddressesProviderRegistry.sol
+++ @aave/core-v3/contracts/interfaces/IPoolAddressesProviderRegistry.sol
@@ -1,12 +1,12 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.8.10;
 
 /**
  * @title IPoolAddressesProviderRegistry
  * @author Aave
  * @notice Defines the basic interface for an Aave Pool Addresses Provider Registry.
- */
+ **/
 interface IPoolAddressesProviderRegistry {
     /**
      * @dev Emitted when a new AddressesProvider is registered.
      * @param addressesProvider The address of the registered PoolAddressesProvider
@@ -29,9 +29,9 @@
 
     /**
      * @notice Returns the list of registered addresses providers
      * @return The list of addresses providers
-     */
+     **/
     function getAddressesProvidersList()
         external
         view
         returns (address[] memory);
@@ -59,13 +59,13 @@
      * @dev The PoolAddressesProvider must not already be registered in the registry
      * @dev The id must not be used by an already registered PoolAddressesProvider
      * @param provider The address of the new PoolAddressesProvider
      * @param id The id for the new PoolAddressesProvider, referring to the market it belongs to
-     */
+     **/
     function registerAddressesProvider(address provider, uint256 id) external;
 
     /**
      * @notice Removes an addresses provider from the list of registered addresses providers
      * @param provider The PoolAddressesProvider address
-     */
+     **/
     function unregisterAddressesProvider(address provider) external;
 }
