Index: lib/solidity-utils/src/contracts/utils/Rescuable.sol
===================================================================
--- lib/solidity-utils/src/contracts/utils/Rescuable.sol
+++ lib/solidity-utils/src/contracts/utils/Rescuable.sol
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.8;
 
-import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {IERC20} from "../oz-common/interfaces/IERC20.sol";
 import {RescuableBase} from "./RescuableBase.sol";
 import {IRescuable} from "./interfaces/IRescuable.sol";
 
 /**
