Index: src/contracts/rewards/RewardsController.sol
===================================================================
--- src/contracts/rewards/RewardsController.sol
+++ src/contracts/rewards/RewardsController.sol
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.10;
 
 import {VersionedInitializable} from "../misc/aave-upgradeability/VersionedInitializable.sol";
-import {SafeCast} from "openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
+import {SafeCast} from "../dependencies/openzeppelin/contracts/SafeCast.sol";
 import {IScaledBalanceToken} from "../interfaces/IScaledBalanceToken.sol";
 import {RewardsDistributor} from "./RewardsDistributor.sol";
 import {IRewardsController} from "./interfaces/IRewardsController.sol";
 import {ITransferStrategyBase} from "./interfaces/ITransferStrategyBase.sol";
