Index: src/contracts/misc/DefaultReserveInterestRateStrategyV2.sol
===================================================================
--- src/contracts/misc/DefaultReserveInterestRateStrategyV2.sol
+++ src/contracts/misc/DefaultReserveInterestRateStrategyV2.sol
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {IERC20} from "../dependencies/openzeppelin/contracts/IERC20.sol";
 import {WadRayMath} from "../protocol/libraries/math/WadRayMath.sol";
 import {PercentageMath} from "../protocol/libraries/math/PercentageMath.sol";
 import {DataTypes} from "../protocol/libraries/types/DataTypes.sol";
 import {Errors} from "../protocol/libraries/helpers/Errors.sol";
