Index: src/contracts/protocol/libraries/logic/LiquidationLogic.sol
===================================================================
--- src/contracts/protocol/libraries/logic/LiquidationLogic.sol
+++ src/contracts/protocol/libraries/logic/LiquidationLogic.sol
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.10;
 
-import {IERC20} from "../../../dependencies/openzeppelin/contracts//IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import {GPv2SafeERC20} from "../../../dependencies/gnosis/contracts/GPv2SafeERC20.sol";
 import {PercentageMath} from "../../libraries/math/PercentageMath.sol";
 import {MathUtils} from "../../libraries/math/MathUtils.sol";
 import {TokenMath} from "../../libraries/helpers/TokenMath.sol";
