===================================================================
--- src/contracts/interfaces/IACLManager.sol
+++ @aave/core-v3/contracts/interfaces/IACLManager.sol
@@ -1,14 +1,14 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.8.10;
 
 import {IPoolAddressesProvider} from "./IPoolAddressesProvider.sol";
 
 /**
  * @title IACLManager
  * @author Aave
  * @notice Defines the basic interface for the ACL Manager
- */
+ **/
 interface IACLManager {
     /**
      * @notice Returns the contract address of the PoolAddressesProvider
      * @return The address of the PoolAddressesProvider
@@ -125,9 +125,9 @@
      */
     function addFlashBorrower(address borrower) external;
 
     /**
-     * @notice Removes an address as FlashBorrower
+     * @notice Removes an admin as FlashBorrower
      * @param borrower The address of the FlashBorrower to remove
      */
     function removeFlashBorrower(address borrower) external;
 
