Index: src/contracts/extensions/stata-token/StataTokenFactory.sol
===================================================================
--- src/contracts/extensions/stata-token/StataTokenFactory.sol
+++ src/contracts/extensions/stata-token/StataTokenFactory.sol
@@ -3,9 +3,9 @@
 
 import {IERC20Metadata} from "openzeppelin-contracts/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import {Initializable} from "openzeppelin-contracts/contracts/proxy/utils/Initializable.sol";
 import {ITransparentProxyFactory} from "solidity-utils/contracts/transparent-proxy/interfaces/ITransparentProxyFactory.sol";
-import {IPool, DataTypes} from "../../../contracts/interfaces/IPool.sol";
+import {IPool} from "../../../contracts/interfaces/IPool.sol";
 import {StataTokenV2} from "./StataTokenV2.sol";
 import {IStataTokenFactory} from "./interfaces/IStataTokenFactory.sol";
 
 /**
