Index: src/contracts/rewards/RewardsDistributor.sol
===================================================================
--- src/contracts/rewards/RewardsDistributor.sol
+++ src/contracts/rewards/RewardsDistributor.sol
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.10;
 
 import {IScaledBalanceToken} from "../interfaces/IScaledBalanceToken.sol";
-import {IERC20Detailed} from "../dependencies/openzeppelin/contracts/IERC20Detailed.sol";
+import {IERC20Metadata} from "openzeppelin-contracts/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import {SafeCast} from "openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
 import {IRewardsDistributor} from "./interfaces/IRewardsDistributor.sol";
 import {RewardsDataTypes} from "./libraries/RewardsDataTypes.sol";
 
@@ -252,9 +252,9 @@
                 _assetsList.push(rewardsInput[i].asset);
             }
 
             uint256 decimals = _assets[rewardsInput[i].asset]
-                .decimals = IERC20Detailed(rewardsInput[i].asset).decimals();
+                .decimals = IERC20Metadata(rewardsInput[i].asset).decimals();
 
             RewardsDataTypes.RewardData storage rewardConfig = _assets[
                 rewardsInput[i].asset
             ].rewards[rewardsInput[i].reward];
@@ -321,8 +321,9 @@
             require(newIndex <= type(uint104).max, "INDEX_OVERFLOW");
             indexUpdated = true;
 
             //optimization: storing one after another saves one SSTORE
+            // forge-lint: disable-next-line(unsafe-typecast)
             rewardData.index = uint104(newIndex);
             rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
         } else {
             rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
@@ -351,8 +352,9 @@
         uint256 rewardsAccrued;
         bool dataUpdated;
         if ((dataUpdated = userIndex != newAssetIndex)) {
             // already checked for overflow in _updateRewardData
+            // forge-lint: disable-next-line(unsafe-typecast)
             rewardData.usersData[user].index = uint104(newAssetIndex);
             if (userBalance != 0) {
                 rewardsAccrued = _getRewards(
                     userBalance,
