===================================================================
--- src/contracts/rewards/interfaces/IRewardsDistributor.sol
+++ @aave/periphery-v3/contracts/rewards/interfaces/IRewardsDistributor.sol
@@ -1,6 +1,6 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.10;
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.10;
 
 /**
  * @title IRewardsDistributor
  * @author Aave
@@ -45,8 +45,18 @@
         uint256 rewardsAccrued
     );
 
     /**
+     * @dev Emitted when the emission manager address is updated.
+     * @param oldEmissionManager The address of the old emission manager
+     * @param newEmissionManager The address of the new emission manager
+     */
+    event EmissionManagerUpdated(
+        address indexed oldEmissionManager,
+        address indexed newEmissionManager
+    );
+
+    /**
      * @dev Sets the end date for the distribution
      * @param asset The asset to incentivize
      * @param reward The reward token that incentives the asset
      * @param newDistributionEnd The end date of the incentivization, in unix time format
@@ -107,20 +117,8 @@
         address reward
     ) external view returns (uint256, uint256, uint256, uint256);
 
     /**
-     * @dev Calculates the next value of an specific distribution index, with validations.
-     * @param asset The incentivized asset
-     * @param reward The reward token of the incentivized asset
-     * @return The old index of the asset distribution
-     * @return The new index of the asset distribution
-     **/
-    function getAssetIndex(
-        address asset,
-        address reward
-    ) external view returns (uint256, uint256);
-
-    /**
      * @dev Returns the list of available reward token addresses of an incentivized asset
      * @param asset The incentivized asset
      * @return List of rewards addresses of the input asset
      **/
@@ -180,13 +178,12 @@
     /**
      * @dev Returns the address of the emission manager
      * @return The address of the EmissionManager
      */
-    function EMISSION_MANAGER() external view returns (address);
+    function getEmissionManager() external view returns (address);
 
     /**
-     * @dev Returns the address of the emission manager.
-     * Deprecated: This getter is maintained for compatibility purposes. Use the `EMISSION_MANAGER()` function instead.
-     * @return The address of the EmissionManager
+     * @dev Updates the address of the emission manager
+     * @param emissionManager The address of the new EmissionManager
      */
-    function getEmissionManager() external view returns (address);
+    function setEmissionManager(address emissionManager) external;
 }
