===================================================================
--- lib/aave-v3-origin/src/contracts/interfaces/IPoolConfigurator.sol
+++ src/contracts/interfaces/IPoolConfigurator.sol
@@ -1,9 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
 import {ConfiguratorInputTypes} from "../protocol/libraries/types/ConfiguratorInputTypes.sol";
-import {IDefaultInterestRateStrategyV2} from "./IDefaultInterestRateStrategyV2.sol";
 
 /**
  * @title IPoolConfigurator
  * @author Aave
