===================================================================
--- src/contracts/protocol/configuration/ACLManager.sol
+++ @aave/core-v3/contracts/protocol/configuration/ACLManager.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: BUSL-1.1
-pragma solidity ^0.8.10;
+pragma solidity 0.8.10;
 
 import {AccessControl} from "../../dependencies/openzeppelin/contracts/AccessControl.sol";
 import {IPoolAddressesProvider} from "../../interfaces/IPoolAddressesProvider.sol";
 import {IACLManager} from "../../interfaces/IACLManager.sol";
@@ -31,9 +31,9 @@
      */
     constructor(IPoolAddressesProvider provider) {
         ADDRESSES_PROVIDER = provider;
         address aclAdmin = provider.getACLAdmin();
-        require(aclAdmin != address(0), Errors.AclAdminCannotBeZero());
+        require(aclAdmin != address(0), Errors.ACL_ADMIN_CANNOT_BE_ZERO);
         _setupRole(DEFAULT_ADMIN_ROLE, aclAdmin);
     }
 
     /// @inheritdoc IACLManager
