===================================================================
--- src/contracts/dependencies/openzeppelin/contracts/IAccessControl.sol
+++ @aave/core-v3/contracts/dependencies/openzeppelin/contracts/IAccessControl.sol
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.10;
+pragma solidity 0.8.10;
 
 /**
  * @dev External interface of AccessControl declared to support ERC165 detection.
  */
