===================================================================
--- lib/aave-v3-origin/src/contracts/protocol/tokenization/AToken.sol
+++ src/contracts/protocol/tokenization/AToken.sol
@@ -3,15 +3,14 @@
 
 import {SafeCast} from "openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
 import {ECDSA} from "openzeppelin-contracts/contracts/utils/cryptography/ECDSA.sol";
 
-import {IERC20} from "../../dependencies/openzeppelin/contracts/IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import {GPv2SafeERC20} from "../../dependencies/gnosis/contracts/GPv2SafeERC20.sol";
 import {VersionedInitializable} from "../../misc/aave-upgradeability/VersionedInitializable.sol";
 import {Errors} from "../libraries/helpers/Errors.sol";
 import {IPool} from "../../interfaces/IPool.sol";
 import {IAToken} from "../../interfaces/IAToken.sol";
-import {IAaveIncentivesController} from "../../interfaces/IAaveIncentivesController.sol";
 import {IInitializableAToken} from "../../interfaces/IInitializableAToken.sol";
 import {ScaledBalanceTokenBase} from "./base/ScaledBalanceTokenBase.sol";
 import {IncentivizedERC20} from "./base/IncentivizedERC20.sol";
 import {EIP712Base} from "./base/EIP712Base.sol";
