Index: src/contracts/protocol/libraries/logic/BorrowLogic.sol
===================================================================
--- src/contracts/protocol/libraries/logic/BorrowLogic.sol
+++ src/contracts/protocol/libraries/logic/BorrowLogic.sol
@@ -2,9 +2,9 @@
 pragma solidity ^0.8.10;
 
 import {GPv2SafeERC20} from "../../../dependencies/gnosis/contracts/GPv2SafeERC20.sol";
 import {SafeCast} from "openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
-import {IERC20} from "../../../dependencies/openzeppelin/contracts/IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import {IVariableDebtToken} from "../../../interfaces/IVariableDebtToken.sol";
 import {IAToken} from "../../../interfaces/IAToken.sol";
 import {IPool} from "../../../interfaces/IPool.sol";
 import {TokenMath} from "../../libraries/helpers/TokenMath.sol";
