Index: src/contracts/protocol/libraries/logic/ValidationLogic.sol
===================================================================
--- src/contracts/protocol/libraries/logic/ValidationLogic.sol
+++ src/contracts/protocol/libraries/logic/ValidationLogic.sol
@@ -1,15 +1,12 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.10;
 
-import {IERC20} from "../../../dependencies/openzeppelin/contracts/IERC20.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import {Address} from "../../../dependencies/openzeppelin/contracts/Address.sol";
 import {GPv2SafeERC20} from "../../../dependencies/gnosis/contracts/GPv2SafeERC20.sol";
-import {IPriceOracleGetter} from "../../../interfaces/IPriceOracleGetter.sol";
 import {IAToken} from "../../../interfaces/IAToken.sol";
 import {IPriceOracleSentinel} from "../../../interfaces/IPriceOracleSentinel.sol";
-import {IPoolAddressesProvider} from "../../../interfaces/IPoolAddressesProvider.sol";
-import {IAccessControl} from "../../../dependencies/openzeppelin/contracts/IAccessControl.sol";
 import {ReserveConfiguration} from "../configuration/ReserveConfiguration.sol";
 import {UserConfiguration} from "../configuration/UserConfiguration.sol";
 import {EModeConfiguration} from "../configuration/EModeConfiguration.sol";
 import {Errors} from "../helpers/Errors.sol";
@@ -18,10 +15,8 @@
 import {DataTypes} from "../types/DataTypes.sol";
 import {ReserveLogic} from "./ReserveLogic.sol";
 import {GenericLogic} from "./GenericLogic.sol";
 import {SafeCast} from "openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
-import {IncentivizedERC20} from "../../tokenization/base/IncentivizedERC20.sol";
-import {MathUtils} from "../math/MathUtils.sol";
 
 /**
  * @title ValidationLogic library
  * @author Aave
